.travis.yml

```diff
@@ -0,0 +1,25 @@
+services:
+  - postgresql
+
+addons:
+  postgresql: "12.2"
```

Comment:
Likely has to do with this version of postgres https://docs.travis-ci.com/user/database-setup/#using-a-different-postgresql-version